### DISCUSSION TITLE
Is the demo website on the latest build ?

### DISCUSSION THREAD

**Discussion by trashcluster** [PhlexPlexico/G5V#40] [General]
http://phlex.avidgamers.me:8080

Hi, i was wondering if the hosted demo website is on the latest build. The team creation process is very buggy and does not work on the demo website and was wondering if i should host it for myself to test the team creation functions.

**Comment by PhlexPlexico** [comment]
Yeah it's available on the latest pull. What's the errors it's giving? It's been working fine for me.

**Comment by PhlexPlexico** [comment]
What exactly is not working? I see that there are two teams currently created :) 
http://phlex.avidgamers.me:8080/teams/114
http://phlex.avidgamers.me:8080/teams/115

**Reply by trashcluster** [reply]
When clicking on the "Create team" menu from the sidebar the "Edit team" button disappears, i managed to click on it by stopping the page loading just before the button disappear. However the "My Teams" and "All Teams" just says no data available

**Reply by PhlexPlexico** [reply]
Also this isn't really an API problem, but a problem with the front end, just as a heads up :)

**Reply by PhlexPlexico** [reply]
As you see, there's no check to see if it's a new team for the button, haha. 

https://github.com/PhlexPlexico/G5V/blob/245fc355b97d8b721689b0b17c9f626902fa796b/src/components/TeamTable.vue#L69

I'll see what I can do. In the meantime if you host your own instance, you can add yourself as a super admin and it should work just dandy.

**Reply by PhlexPlexico** [reply]
There, adjusted the if statement to catch that as well, thanks for testing that out! It should be good now.

**Reply by trashcluster** [reply]
Awesome

**Comment by PhlexPlexico** [comment]
Ah I see. It's a `v-if` block that's failing to load the content. I'll see what I can do to adjust that so it loads properly.
